lib/getUniqueMatch.js

```diff
@@ -2,19 +2,25 @@
 var Cesium = require('cesium');
 var defined = Cesium.defined;
 
+module.exports = getUniqueMatch;
+
 /** Generic regex search over array of String. (Requires global regex)
```

Comment:
It requires the `g` regex flag, so it looks like `/.../g`. Otherwise it will loop forever.